### ISSUE TITLE
Refresh "Introduction to Library Carpentry"

### ISSUE THREAD

**Issue by sharilaster** [LibraryCarpentry/lc-overview#34]
In responding to #33 I had a closer look at episode 2, "Introduction to Library Carpentry – Teaching Data Science Skills." I think it's not clear to me what the purpose and audience for this episode is. As currently presented it looks like various notes gathered over time about LC's history and its potential value to learners. 

I realize that lc-overview is a bit stitched together from various places, and that the episode is not a part of the curriculum, but I do think it could be worthwhile to consider if there's a need for an overview beyond what is on the LC website, and if so, what the audience and narrative should be.

**Comment by libcce** [comment]
Hi @sharilaster the episode comes from a slide deck I often delivered to university library administrators or as part of webinars. I often had the difficult task of also presenting to both librarians and researchers, so the audience lines might be a bit blurred. I'm now foggy on why this episode was created, possibly to move away from the slide deck and have something that could be improved like other lesson episodes (but available to anyone who needed to provide an overview, as a place to start, in an initial intro to LC workshop)? No doubt it still needs work as it follows the slide deck format largely. I can bring this up in our LC Advisory Group Mtg on Nov 24.

**Comment by sharilaster** [comment]
@libcce Sounds great, thank you! Happy to help with revisions based on their decision.

**Comment by libcce** [comment]
@sharilaster LC Advisory Group is focusing on updating the website as our next action. The Carpentries will be looking into their roadmap for the website(s) in general.

**Comment by tobyhodges** [comment]
I wonder if a (short term) compromise could be found by moving this page to Extras instead of Episodes? (And adjusting the redirect from the librarycarpentry.org accordingly.) It would separate this material from the curriculum itself, but still associate it closely with the lc-overview lesson. But maybe you will prefer to move this material away from the lesson repository altogether?

**Comment by libcce** [comment]
If moved to Extras, it might lose some visibility. I guess I wonder from the usage, are people using it, is it useful? Before we move it. But also, we need to consider the overview page and how that is structured as well. 

One proposal from @jt14den was to move back to using slides. I just wonder about the slides, they exist and we can link to them, but people cannot contribute back to them and they can grow stale. 

I believe the guidance from LC Advisory Group was update the website, include the necessary overview information there, point to this overview information from the Intro to LC lesson? We have a separate question to The Carpentries, what is the roadmap for updating the Carpentries Lesson Program Websites, so SWC, DC, LC? This may impact our work?

**Comment by kristindawn** [comment]
I am getting ready to teach Library Carpentry in about a month and wanted to check in on this issue. Having the Library Carpentry Workshop Overview as a landing page for the Intro lesson is confusing, especially since it is labelled episode 2 and we go to Jargon Busting as episode 3 so it looks like we jump episodes. Can we have a landing page that looks like the rest of them, with the schedule? I also think we should move the Library Carpentry Workshop Overview to something separate, it is a lot of information at the beginning where my experience is people are already a little nervous about being in the session. Maybe an intro page that just talks about why we need to talk about data in libraries?

**Comment by libcce** [comment]
@kristindawn I was thinking about this recently (since I had to do the what is LC intro for a site) whether the Intro to LC/the Overview can be more of a bulleted list of topics to possibly cover? At the moment it is slides converted to .md which is not exactly an optimal approach. Plus, people would have greater flexibility with a bulleted option, to tailor an intro to their local community? Or we can remove it entirely and point to intros that we can find online that various groups have done?

**Comment by kristindawn** [comment]
@libcce I like the idea of a bulleted list. One of the things I hear most about Library Carpentry is "Great, I can use software, but so what?", so having some examples of why data matters to library workers and how you can chain the skills (Unix->Git for workflows, Excel+OpenRefine+(SQL/Python) for analysis) would be a great place to start. It is also a great way to start the Jargon Busting discussion that comes next ("Here are some things you could do, now think about what you don't know about them"). Can I help get this started somehow?

**Comment by libcce** [comment]
Yes @kristindawn chaining the tools/skills together to help learners make those connections re: why this is important is really needed. If you want to get this started, that would be super helpful! BTW, there have been some cases where people have expressed interest in linking this all to FAIR and/or reproducibility as well. But in many ways, that slidedeck can be probably weeded down to main areas to cover including what you just brought up. This is the most recent one I gave at CDC https://twitter.com/libcce/status/1392435087288020992?s=20. Thanks @kristindawn for taking a crack at it. And can help you when you've got a draft 😄

**Comment by kristindawn** [comment]
@libcce  I made a first pass of refreshing the intro episode of this lesson. None of it is dear to me so feel free to hack away at it. I based it on some things I've experienced and heard teaching it and how I talk about data management in my place-of-work.

**Comment by libcce** [comment]
Thanks @kristindawn I've shared your PR w/ the LC Advisory Group https://github.com/LibraryCarpentry/lc-overview/pull/35. I might be too close to this so I'm hoping someone on the Advisory Group is interested in looking at this with fresh eyes 👀

**Comment by morskyjezek** [comment]
Hi @libcce & @kristindawn - did you receive any feedback from the advisory group? This has been hanging out here for a while and is reflected in the current proposal in #35 . Any updates?

**Comment by morskyjezek** [comment]
All, I've closed the PR due to the upcoming transition work. But I think this is still worth discussing in relation to the revision of the lesson.
